modules/storage-page-memory/src/main/java/org/apache/ignite/internal/storage/pagememory/mv/MvPageTypes.java

```diff
@@ -30,9 +30,6 @@ public interface MvPageTypes {
     /** Version chain tree leaf page IO type. */
     short T_VERSION_CHAIN_LEAF_IO = 11;
 
-    /** Row version data page IO type. */
-    short T_ROW_VERSION_DATA_IO = 12;
```

Comment:
What if we re-assign these numbers, to make them more meaningful? This gap makes no sense now